deadass/templates/dbform.html

```diff
@@ -12,6 +12,7 @@
     <h1 class="display-4 mt-4">Create New Database</h1>
       <hr class="my-4">
       <div class="container">
+        <p style="text-align: center">(Database name must be alphanumeric!)</p>
```

Comment:
Can you put it like this?
`<div class="alert alert-warning" role="alert">{{ message }}</div>`